src/main/java/com/dev/cinema/Main.java

```diff
@@ -1,18 +1,43 @@
 package com.dev.cinema;
 
 import com.dev.cinema.lib.Injector;
+import com.dev.cinema.model.CinemaHall;
 import com.dev.cinema.model.Movie;
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.service.CinemaHallService;
 import com.dev.cinema.service.MovieService;
+import com.dev.cinema.service.MovieSessionService;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
 
 public class Main {
     private static Injector injector = Injector.getInstance("com.dev.cinema");
 
     public static void main(String[] args) {
+
+        MovieService movieService =
+                (MovieService) injector.getInstance(MovieService.class);
         Movie movie = new Movie();
-        movie.setTitle("Fast and Furious");
-        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
+        movie.setTitle("Some title");
+        movie.setDescription("Some description");
         movieService.add(movie);
-
-        movieService.getAll().forEach(System.out::println);
+        System.out.println(movieService.getAll());
+        CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(100);
+        cinemaHall.setDescription("Some description");
+        cinemaHallService.add(cinemaHall);
+        System.out.println(cinemaHallService.getAll());
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(movie);
+        movieSession.setCinemaHall(cinemaHall);
+        movieSession.setSessionTime(LocalDateTime
+                .of(2021, 2, 2, 10, 30));
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
+        movieSessionService.add(movieSession);
+        movieSessionService.findAvailableSessions(1L, LocalDate
+                .of(2021, 2, 2)).forEach(System.out::println);
```

Comment:
what gets printed out? can you please share the screenshot?

Reply:
![image](https://user-images.githubusercontent.com/39372122/106585335-e264a300-654f-11eb-8f77-c1f8b5039996.png)